src/servers/launcher/main.cpp

```diff
@@ -0,0 +1,849 @@
+#include <libpq-fe.h>
+
+#include <array>
+#include <cerrno>
+#include <chrono>
+#include <cstdint>
+#include <cstring>
+#include <expected>
+#include <iomanip>
+#include <iostream>
+#include <netinet/in.h>
+#include <ranges>
+#include <signal.h>
+#include <stdexcept>
+#include <sstream>
+#include <string>
+#include <sys/epoll.h>
+#include <sys/wait.h>
+#include <thread>
+#include <unistd.h>
+#include <utility>
+#include <variant>
+
+using namespace std::literals;
+using notManyMilliseconds = std::chrono::duration<int, std::milli>;
+using PacketBuffer = std::array<uint8_t, 65536>;
+
+static constexpr const auto RETRY_DELAY{1s};
+static constexpr const uint16_t PORT{32232};
+static constexpr const uint8_t CABOOSE{108};
+
+struct UUID {
+  std::array<const uint8_t, 16> uuid;
```

Comment:
I’m not sure using a 16-octet array is the best way to define a UUID. I know there are not *standard* 128-bit types ([yet!](https://wg21.link/p3140)), but every compiler already has one. It might make more sense to use that, although you might have to be cautious of endianness.

Reply:
So the server executables will all be compiled for my machines, so using non-standard features isn't a big deal, but the game client will be running out there in the wild, and I imagine it will want to share UUIDs with the servers.  Am I walking down a silly path with this train of thought?